corsMiddleware.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept, auth-token');
```

Comment:
The '*' in 'Access-Control-Allow-Origin' allows all origins. Consider using a whitelist of trusted domains to restrict access. This increases security by preventing unwanted domains from making requests.